force-app/main/default/lwc/gePaymentGatewayManagement/gePaymentGatewayManagement.js

```diff
@@ -52,22 +57,36 @@ export default class GePaymentGatewayManagement extends LightningElement {
     isSystemAdmin;
     hasAccess;
 
-    CUSTOM_LABELS = { messageLoading, insufficientPermissions, commonAdminPermissionErrorMessage };
+    @track isGatewayAssignmentEnabled;
```

Comment:
We do not need the `@track` annotation here since it is just a `boolean` and not an object that has properties that we need to track their reactivity.

Reply:
updated